### PR TITLE
issue #13, #16, and #17

### LINKED ISSUE #13: Implement system language detection
Either Italian or English

### LINKED ISSUE #16: Update background colors

### LINKED ISSUE #17: Add svg logo to the header
With appropriate dimensions.

### PR DESCRIPTION
**Pull request by itIsJim**
svg logo added and adjusted/ css background gradient/ language detection implementation

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -38,19 +38,21 @@
...
 }
 
-/*--------- CSS animated Background ----------*/
+
+
+/*--------- CSS Animated Background ----------*/
 
 
 body {
   height: 100vh;
```
Comment (e-caste) [inline src/App.css]: There's an issue on mobile and I think this is the line at fault: the background "repeats" every height of the phone while scrolling, here's what it looks like:

https://user-images.githubusercontent.com/48513706/144081901-9961508c-ad92-4622-b56e-30207398e9b3.MP4

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (e-caste) [inline .idea/inspectionProfiles/Project_Default.xml]: You should add the following line in the root .gitignore file:  
`.idea/`

And you should remove these files, either by `git reset --soft HEAD~1` one by one, and then committing again, or by deleting these files in a second commit. The best would be to simply remove this commit from the log.

**Review by e-caste — CHANGES_REQUESTED** [review]

File: .idea/inspectionProfiles/Project_Default.xml (hunk 2 above)
Comment (itIsJim) [reply .idea/inspectionProfiles/Project_Default.xml]: You mean the entire file .idea should be removed?

File: src/App.css (hunk 1 above)
Comment (itIsJim) [reply src/App.css]: ok I will fix it

**Comment by itIsJim** [comment]
I will fix both of the problems and do a commit

**Review by e-caste — APPROVED** [review]

**Pull request merged**